torchci/.env.example

```diff
@@ -19,4 +19,4 @@ GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
 AUTH_SECRET=
 JWT_SECRET=
-NEXTAUTH_URL=
\ No newline at end of file
+# NEXTAUTH_URL= # uncomment this line if you want to enter a custom url for the next-auth module
```

Comment:
I'm using `NEXTAUTH_URL=http://localhost:3000` and I guess most of us will also use the same value, so may be you want to have that as a default value here or may be mention it as part of the comment

Reply:
nice, I'll change it to that instead